frontend/src/views/account/ListExtensions.vue

```diff
@@ -1,6 +1,175 @@
+<template>
+    <header-container title="My Apps" />
+
+    <main-container v-if="!extensionStore.isLoading">
+        <div
+            v-if="extensionStore.extensions.length === 0"
+            class="text-center"
+        >
+            <svg
+                class="mx-auto h-12 w-12 text-gray-400 dark:text-neutral-500"
+                fill="none"
+                viewBox="0 0 24 24"
+                stroke="currentColor"
+                aria-hidden="true"
+            >
+                <path
+                    vector-effect="non-scaling-stroke"
+                    stroke-linecap="round"
+                    stroke-linejoin="round"
+                    stroke-width="2"
+                    d="M9 13h6m-3-3v6m-9 1V7a2 2 0 012-2h6l2 2h6a2 2 0 012 2v8a2 2 0 01-2 2H5a2 2 0 01-2-2z"
+                />
+            </svg>
+            <h3 class="mt-2 font-medium">
+                No Apps
+            </h3>
+            <p class="mt-1 text-gray-500 dark:text-neutral-500">
+                Get started by adding your first Shop.
+            </p>
+            <div class="mt-6">
+                <router-link
+                    to="/account/shops/new"
+                    class="btn btn-primary group flex items-center"
+                >
+                    <icon-fa6-solid:plus
+                        class="-ml-1 mr-2 h-4 w-4 opacity-25 group-hover:opacity-50"
+                        aria-hidden="true"
+                    />
+                    Add Shop
+                </router-link>
+            </div>
+        </div>
+
+        <template v-else>
+            <input
```

Comment:
Search no longer works
<img width="474" alt="Bildschirmfoto 2024-03-09 um 18 39 24" src="https://github.com/FriendsOfShopware/shopmon/assets/412142/cb8d6004-f461-4b2c-90bc-d97d5b2166d7">
<img width="775" alt="Bildschirmfoto 2024-03-09 um 18 39 55" src="https://github.com/FriendsOfShopware/shopmon/assets/412142/2c263327-a99e-496d-a17c-1afac03c15ee">

Reply:
Fixed that by marking name as searchable